docs/versioned_docs/version-3.6/manual/running.md

```diff
@@ -49,6 +49,12 @@ On Windows, OpenRefine can also be run by using the file `refine.bat` in the pro
 If you call `refine.bat` from the command line, you can [start OpenRefine with modifications](#starting-with-modifications). 
 If you want to modify the way `refine.bat` opens through double-clicking or using a shortcut, you can edit the `refine.ini` file. 
 
+:::caution Did you get a JAVA_HOME error?
+“Error: Could not find the ‘java’ executable at ‘’, are you sure your JAVA_HOME environment variable is pointing to a proper java installation?”
+
+If you see this error, you need to [install and configure a JDK package](installing#java), including setting up `JAVA_HOME`.
```

Comment:
```suggestion
If you see this error, you need to [install and configure Java](installing#java), including setting up `JAVA_HOME`.
```
JDK is a technical term so I would rather say just "Java" to keep things simple.